src/app/events/[slug]/page.tsx

```diff
@@ -71,6 +72,10 @@ export default function EventEntry({ params }: EventProps) {
         />
       </div>
 
+      {schedule && schedule.days && schedule.days.length > 0 && (
+        <ScheduleSection schedule={schedule} />
+      )}
+
```

Comment:
what do you think of extracting conditions in variables? So we have something like this 

```
  const eventHasSchedule = schedule && schedule.days.length > 0
  const eventHasSpeakers = speakers && speakers.length > 0
  const eventHasSponsors = sponsors && Object.keys(sponsors).length > 0
```

...

`      {eventHasSchedule && <ScheduleSection schedule={schedule} />}
`

Reply:
I like it a lot.